model/version.go

```diff
@@ -399,11 +399,11 @@ func getMostRecentMainlineCommit(projectId string) (*Version, error) {
 	res := []Version{}
 
 	if err := db.Aggregate(VersionCollection, pipeline, &res); err != nil {
-		return nil, errors.Wrapf(err, "error aggregating versions")
+		return nil, errors.Wrapf(err, "aggregating versions")
 	}
 
 	if len(res) == 0 {
-		return nil, errors.Errorf("no mainline commit found for project '%v'", projectId)
+		return nil, errors.Errorf("no mainline commit found for project '%s'", projectId)
```

Comment:
`could not find mainline commit for project...`

Reply:
Missed this.

Reply:
Fixed.